### PR TITLE
More in depth configuration of SElinux and iptables for 1.5

### PR DESCRIPTION
**Pull request by Mreyna3**
Modified the 1.5 build instructions, to display more in depth detail surrounding :

Disabling SELinux
-Opening iptables
Instructions pertain to centOS/RHEL 6/7

### CODE REVIEW

File: source/installation/modify-system-security.rst [hunk 1]
```diff
@@ -14,6 +14,46 @@ Modify System Security
...
+         Mode from config file:          enforcing
+         Policy MLS status:              enabled
+         Policy deny_unknown status:     allowed
+         Max kernel policy version:      31
+
+
+   RHEL/CentOS 6/7
+      .. code-block:: sh
+
+         $ sudo chmod -R 757 /etc/selinux/config
```
[the remark this answers is not in the document]
Comment (Mreyna3) [reply source/installation/modify-system-security.rst]: I removed the block you suggested. And added setenforce 0 as method for disabling the SELinux.

File: source/installation/modify-system-security.rst [hunk 2]
```diff
@@ -32,27 +23,52 @@ Modify System Security
...
+      If SELinux is not disabled on the web server host machine this can lead
+      to permission errors when running the web applications. SELinux may also
+      need to be disabled on the web server if you want to take advantage of
+      certain software in your web applications such as being able to access a
+      Lustre file system.
 
-         $ sudo chmod -R 757 /etc/selinux/config  
 
-   Using your favorite text editor open the /etc/selinux/config file and set the SELINUX mode to disabled 
+   **Option 1: Disable SELinux Temporarily**
```
Comment (MorganRodgers) [inline source/installation/modify-system-security.rst]: Using `setenforce 0` should not be presented as a separate option from editing `/etc/selinux/config`. They should be used together.

File: source/installation/modify-system-security.rst [hunk 3]
```diff
@@ -32,27 +23,52 @@ Modify System Security
...
+               #       enforcing - SELinux security policy is enforced.
+               #       permissive - SELinux prints warnings instead of enforcing.
+               #       disabled - No SELinux policy is loaded.
+               SELINUX=disabled
+               # SELINUXTYPE= can take one of these two values:
+               #       targeted - Targeted processes are protected,
+               #       mls - Multi Level Security protection.
+               SELINUXTYPE=targeted
+
+      #. Reboot the OS to save the changes
```
Comment (MorganRodgers) [inline source/installation/modify-system-security.rst]: Rebooting is not necessary to disable SELinux if the user runs `setenforce 0`.

File: source/installation/modify-system-security.rst [hunk 4]
```diff
@@ -73,7 +89,7 @@ Modify System Security
 
    RHEL/CentOS 6
       .. code-block:: sh
-
```
Comment (MorganRodgers) [inline source/installation/modify-system-security.rst]: In general you should avoid whitespace changes that are not relevant to the task at hand to keep the diffs clean. In this case you remove a tab which we did not need, so no harm done.

**Comment by ericfranz** [comment]
Replaced by https://github.com/OSC/ood-documentation/pull/217

**Pull request closed without merging**